src/main/java/at/hannibal2/skyhanni/config/features/dev/DevConfig.kt

```diff
@@ -127,11 +127,14 @@ class DevConfig {
     @ConfigEditorBoolean
     var numberFormatOverride: Boolean = false
 
-    // TODO reenable the setting once the hypixel mod api works fine
-//     @Expose
-//     @ConfigOption(name = "Use Hypixel Mod API", desc = "Use the Hypixel Mod API for better location data.")
-//     @ConfigEditorBoolean
-//     var hypixelModApi: Boolean = true
+    @Expose
+    @ConfigOption(
+        name = "Use Hypixel Mod API",
+        desc = "Use the Hypixel Mod API for better location data.\n" +
+            "§cNote: this setting may be disabled remotely, and will only work on beta versions."
+    )
+    @ConfigEditorBoolean
+    var hypixelModApi: Boolean = false // should this be default on or off?
```

Comment:
first keep it off per default, so that we (the devs) can enable it for our sessions and see if we notice problems. we can enable this per default later again. (and way later remove the option entirely as we phase off from scoreboard/tab list)